README.md

```diff
@@ -2,10 +2,14 @@
 
 ## Setup
 
-1. Install PyTorch separately (for local models). You need torch for most experiments and torchvision for the MNIST toy model.
-> See https://pytorch.org/get-started/locally/
+To setup, simply run `pip install .` from project root
+
+### Development
+
+You can install in dev mode to include additional dependencies for data visualization etc.
 
-2. Install the requirements
 ```bash
-pip install -r requirements.txt
-```
\ No newline at end of file
+pip install ".[dev]"
+# Or for editable mode
+pip install -e ".[dev]"
```

Comment:
You probably should tell developers to install only in editable mode, so you can remove the first one here.